packages/components/src/components/button/button.hbs

```diff
@@ -28,27 +28,27 @@
   {{ carbon-icon 'Add16' class=(add @root.prefix '--btn__icon') }}
 </button>
 {{#if hasIconOnly}}
-  <button
-    class="{{@root.prefix}}--btn {{@root.prefix}}--btn--{{variant}} {{@root.prefix}}--btn--icon-only {{@root.prefix}}--btn--icon-only--bottom {{#if small}} {{@root.prefix}}--btn--sm{{/if}}{{#if field}} {{@root.prefix}}--btn--field{{/if}}"
-    aria-label="Add">
-    {{ carbon-icon 'Add16' class=(add @root.prefix '--btn__icon') }}
-  </button>
+<button
+  class="{{@root.prefix}}--btn {{@root.prefix}}--btn--{{variant}} {{@root.prefix}}--btn--icon-only {{@root.prefix}}--tooltip__trigger {{@root.prefix}}--tooltip--a11y {{@root.prefix}}--tooltip--bottom {{@root.prefix}}--tooltip--align-center {{#if small}} {{@root.prefix}}--btn--sm{{/if}}{{#if field}} {{@root.prefix}}--btn--field{{/if}}">
+  <span class="{{@root.prefix}}--assistive-text">Add</span>
+  {{ carbon-icon 'Add16' class=(add @root.prefix '--btn__icon') }}
+</button>
 {{/if}}
 {{#if field}}
-  <!-- The markup below is for demonstration purposes only -->
-  <div class="example-input-button-pairing">
-    <div class="{{@root.prefix}}--form-item">
-      <label for="text-input-3" class="{{@root.prefix}}--label">Text Input label</label>
-      <input id="text-input-3" type="text"
-        class="{{@root.prefix}}--text-input{{#if light}} {{@root.prefix}}--text-input--light{{/if}}"
-        placeholder="Optional placeholder text">
-    </div>
-    <div class="example-button-wrapper">
-      <button class=" {{@root.prefix}}--btn {{@root.prefix}}--btn--{{variant}}{{#if small}}
+<!-- The markup below is for demonstration purposes only -->
+<div class="example-input-button-pairing">
+  <div class="{{@root.prefix}}--form-item">
+    <label for="text-input-3" class="{{@root.prefix}}--label">Text Input label</label>
+    <input id="text-input-3" type="text"
+      class="{{@root.prefix}}--text-input{{#if light}} {{@root.prefix}}--text-input--light{{/if}}"
+      placeholder="Optional placeholder text">
+  </div>
+  <div class="example-button-wrapper">
+    <button class=" {{@root.prefix}}--btn {{@root.prefix}}--btn--{{variant}}{{#if small}}
       {{@root.prefix}}--btn--sm{{/if}}{{#if field}} {{@root.prefix}}--btn--field{{/if}}" {{#if danger}}
-        aria-label="danger" {{/if}} type="button">
-        Button
-      </button>
-    </div>
+      aria-label="danger" {{/if}} type="button">
```

Comment:
FYI, if you add an `aria-label` to a button the SR doesn't actually read the contents of them button so the user would never hear `Button`.  I had a chat with @snidersd about this and she said that for danger there really isn't a way to denote this variant generically to the user but we be following best practices for content design and making sure the label of the button is clear to the user.

Reply:
thanks for the info, this was just carried over from our existing button examples. should we open a new PR to remove that `aria-label` in all of the examples?

Reply:
I agree that it's beyond the scope of this PR and should be moved to a separate issue.